server/models/service-instance.js

```diff
@@ -66,6 +67,32 @@ module.exports = function(ServiceInstance) {
     });
   });
 
+  function recordInstanceInfo(instanceId, instInfo, callback) {
+    ServiceInstance.findById(instanceId, function(err, instance) {
+      if (err) return callback(err);
+      instance.currentDeploymentId = instInfo.commitHash;
+      instance.startTime = new Date();
+      instance.started = true;
+      instance.applicationName = instInfo.appName;
+      instance.npmModules = instInfo.npmModules;
+      instance.PMPort = instInfo.PMPort;
+      instance.containerVersionInfo = {
+        os: {
+          platform: os.platform(),
```

Comment:
Can these runtime details like OS and startTime be passed in as part of `instInfo` instead? I'm picturing a future bug where someone's cluster is left on an older version of Node because all their apps appear to be running in the exact same environment as their cluster controller.